android/travis_build_all.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+set -e
+
+#
+# See: https://boinc.berkeley.edu/trac/wiki/AndroidBuildClient
+#
+
+# Script to compile everything BOINC needs for Android
+
+./buildAndroidBOINC-CI.sh --cache_dir "$ANDROID_TC" --build_dir "$BUILD_DIR" --silent --ci --arch arm
```

Comment:
`ANDROID_TC` and `BUILD_DIR` are undefined here

Reply:
Yes. it like was inside the travis. and it like the build_all.sh.
Just recently I define BUILD_DIR in build_all.sh to be the cache folder. but it not good for the CI.
Inside buildAndroidBOINC-CI.sh it define the ANDROID_TC and BUILD_DIR like always.

Reply:
If you look on the build_all.sh script. you will see it copy past from there, just without the BUILD_DIR that i define and also more parameters for the ci.

Reply:
I mean, the real command that will be run is next:
`./buildAndroidBOINC-CI.sh --cache_dir "" --build_dir "" --silent --ci --arch arm`
because these variables are not defined

Reply:
Yes. I understand. It the same without run with these params: `--cache_dir` `--build_dir`.
You want me to remove it.
I just wanted to be the same like `build_all.sh` script.

Reply:
If in the future we will want to define `ANDROID_TC `or `BUILD_DIR` in the script, it will be more easy.